tutorials/3d/csg_tools.rst

```diff
@@ -126,21 +127,21 @@ Create a ``CSGBox`` and adjust its dimension for the bed. Create another ``CSGBo
 
 .. image:: img/csg_bed_mat.png
 
-We will create another ``CSGCombiner`` and name it ``pillow`` as the child of  ``bed``. The scene tree should look like this:
+We will create another ``CSGCombiner``\ , named ``pillow``\ , as the child of  ``bed``. The scene tree should look like this:
```

Comment:
Commas don't need to be escaped after <pre>``</pre> inline code markup, only alphanumeric characters it seems. No idea why, it's a weird aspect of the reStructuredText format :)

Reply:
@akien-mga : Oops, should have read further down the page before asking stupid questions. :grin: Thanks for that titbit.